app/src/main/java/com/nononsenseapps/feeder/ui/compose/feedarticle/ArticleScreen.kt

```diff
@@ -459,6 +490,26 @@ fun ArticleContent(
     }
 }
 
+@Composable
+private fun SummarySection(summary: OpenAISummaryState) {
+    OutlinedCard(
+        modifier = Modifier.fillMaxWidth(),
```

Comment:
```suggestion
                modifier = Modifier.width(LocalDimens.current.maxReaderWidth),
```

Fixes width on tablets